docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -1,11 +1,11 @@
-# Retry Endpoint
+# Retry Endpoint Design (Round 2!)
 
 Date: 2021-05-19
 
 
 ## Motivation for the Endpoint:
 
-The current requirement for the Batch/Bulk UI is to be able to retry jobs that have either "errored" out, or were cancelled.
+The current requirement for the Batch/Bulk UI is to be able to retry jobs that have either "errored" out, or were terminated.
 The UI allows you to retry either single jobs, or multiple jobs, and saves you from having to cancel and resubmit each job individually,
 which is not really possibly with the UI anyway.
```

Comment:
*possible

And it IS possible, just not implemented, and would be a huge pain. Selecting a number of jobs and hitting Retry is the way to go.

Reply:
Ok I will change the wording to "Which is not implemented in the UI"